library/src/main/java/com/google/android/exoplayer/MediaCodecTrackRenderer.java

```diff
@@ -166,6 +181,7 @@ private static String getDiagnosticInfoV21(Throwable cause) {
   private boolean outputStreamEnded;
   private boolean waitingForKeys;
   private boolean waitingForFirstSyncFrame;
+  private boolean hasQueuedOneInputBuffer;
```

Comment:
Small thing: Can we call it hasQueuedInputBuffer? The current name kind of suggests it might be false if we've queued more than one.